model/godleyTableWindow.cc

```diff
@@ -483,15 +483,17 @@ namespace minsky
     motionRow=motionCol=-1;
     if (selectedRow==0)
       {
-        if (c>0 && c!=selectedCol)
+        if (c>0 && selectedCol!=0 && c!=selectedCol)      // Cannot move Intitial Conditions cell from col 0 to one with a "NoAssetClass" designation. For ticket 1064/1066
           godleyIcon->table.moveCol(selectedCol,c-selectedCol);
       }
     else if (r>0 && selectedCol==0)
       {
-        if (r!=selectedRow && !godleyIcon->table.initialConditionRow(selectedRow) && !godleyIcon->table.initialConditionRow(r))  // Cannot move Intitial Conditions row. For ticket 1064
+		// Cannot move Intitial Conditions row. For ticket 1064. Also cannot move last row down, i.e., button widgets have to be used to create new rows, otherwise an error occurs. For ticket 1066.  
+        if (r!=selectedRow && r<int(godleyIcon->table.rows()) && !godleyIcon->table.initialConditionRow(selectedRow) && !godleyIcon->table.initialConditionRow(r))  
```

Comment:
I'm puzzled about why you don't bounds check selectedRow as well. I must admit, I can't trigger out of bounds on row in the existing code, so its probably checked elsewhere, but there is no harm in being defensive here.

Reply:
Oops, I should have added that in there. It will be fixed post haste!